Assets/MixedRealityToolkit.Tests/EditModeTests/Experimental/MigrationToolTest.cs.meta

```diff
@@ -0,0 +1,11 @@
+fileFormatVersion: 2
+guid: 5558e3726716dc5448371b891f917081
+MonoImporter:
+  externalObjects: {}
```

Comment:
nit: file name should be MigrationToolTest**s**.cs.meta?

Reply:
think I didn't added the new meta, and the old one is still there. I will delete this file, as the meta will be generated auto right?